Domain/ProductManagement/InventoryUI.cs

```diff
@@ -0,0 +1,181 @@
+﻿namespace LenaInventoryManagementSystem;
+
+public class InventoryUI 
```

Comment:
all the methods here can be static and the class, you don't have props/state in this class, so no need to create a new object from it to invoke its methods ;)